extra/mariabackup/encryption_plugin.cc

```diff
@@ -72,7 +72,14 @@ static std::string get_encryption_plugin_from_cnf()
       plugin_load = line + 12;
       // remote \n at the end of string
       plugin_load.resize(plugin_load.size() - 1);
-      break;
+    }
+
+    if (strncmp(line, "innodb_encrypt_tables=", 21) == 0)
+    {
+      char *val = line + 22;
+      if (strncmp(val, "ON", 2) == 0)
+        srv_encrypt_tables= 1;
+      else srv_encrypt_tables= 0;
```

Comment:
Is there any Boolean parsing function that we could invoke here? It would seem that `encryption_plugin_backup_init()` is writing `0` or `1`, which also are valid values. Other valid values would be `on`, `off`, `Off`, and I could imagine that `yes`, `no`, `true`, `false` might work as well.